steward/src/commands/fund_cellar.rs

```diff
@@ -15,11 +15,14 @@ use steward_abi::cellar_uniswap::*;
 
 /// Command to fund Cellars
 #[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Fund Cellar.\n This command funds the required Cellar by loading the Cellar config. \n It takes the amounts for the Cellar pool pair."
+)]
 pub struct FundCellarCmd {
-    #[clap(short = 'i', long)]
-    pub cellar_id: u32,
```

Comment:
I don't think we can delete this. The command is just retrieving the first element (`cellars.get(0)`) which means you can't really pick what cellar you're funding. I think the likely intent here is that we take `cellar_id` as a string and find the config that matches (just like we are doing when handling allocation messages).